PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionTransparencyFormat.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Drawing;
+using Microsoft.Office.Core;
+using PowerPointLabs.ActionFramework.Common.Log;
+using Graphics = PowerPointLabs.Utils.GraphicsUtil;
+using Shape = Microsoft.Office.Interop.PowerPoint.Shape;
+
+namespace PowerPointLabs.SyncLab.ObjectFormats
+{
+    class ReflectionTransparencyFormat: Format
+    {
+        public override bool CanCopy(Shape formatShape)
+        {
+            return formatShape.Reflection.Type != MsoReflectionType.msoReflectionTypeNone;
+        }
+
+        public override void SyncFormat(Shape formatShape, Shape newShape)
+        {
+            if (!Sync(formatShape, newShape))
+            {
+                Logger.Log(newShape.Type + " unable to sync ReflectionTransparency format");
+            }
+        }
+
+        public override Bitmap DisplayImage(Shape formatShape)
+        {
+            return SyncFormatUtil.GetTextDisplay(
+                Math.Round(formatShape.Reflection.Transparency * 100).ToString() + "%",
```

Comment:
Do you think it is needed to explain why need to multiply by 100?

Reply:
Hmm.. the docs do say that it's a float between 0 & 1.
And we're displaying as a percentage so.. maybe not?

Reply:
Oh if the docs mention then it's fine then!

Reply:
Cancel that, the docs are a bit ambiguous. I'll update that in a bit